themes/bakke/bakke.theme.sh

```diff
@@ -16,7 +16,15 @@ function _omb_theme_PROMPT_COMMAND() {
     #PS1="${_omb_prompt_bold_teal}$(scm_char)${_omb_prompt_green}$(scm_prompt_info)${_omb_prompt_purple}$(_omb_prompt_print_ruby_env) ${_omb_prompt_olive}\h ${_omb_prompt_reset_color}in ${_omb_prompt_green}\w ${_omb_prompt_reset_color}\n${_omb_prompt_green}→${_omb_prompt_reset_color} "
     #PS1="\n${_omb_prompt_purple}\h: ${_omb_prompt_reset_color} ${_omb_prompt_green}\w\n${_omb_prompt_bold_teal}$(scm_char)${_omb_prompt_green}$(scm_prompt_info) ${_omb_prompt_green}→${_omb_prompt_reset_color} "
     #PS1="\n${_omb_prompt_teal}\h: ${_omb_prompt_reset_color} ${_omb_prompt_olive}\w\n${_omb_prompt_brown}$(scm_char)${_omb_prompt_brown}$(scm_prompt_info) ${_omb_prompt_green}→${_omb_prompt_reset_color} "
-    PS1="\n${_omb_prompt_teal}\h: ${_omb_prompt_reset_color} ${_omb_prompt_olive}\w ${_omb_prompt_green}$(scm_prompt_info)\n${_omb_prompt_reset_color}→ "
+
+    local python_venv
+    _omb_prompt_get_python_venv
+
+    PS1="\n$python_venv ${_omb_prompt_teal}\h: ${_omb_prompt_reset_color} ${_omb_prompt_olive}\w ${_omb_prompt_green}$(scm_prompt_info)\n${_omb_prompt_reset_color}→ "
```

Comment:
> `...$python_venv ...`

In this way, an extra space is added at the beginning of the prompt even when there are no python environments. Can you instead add a space at the end of `OMB_PROMPT_{VIRTUALENV,CONDAENV}_FORMAT`?

I'm not sure if there is such a realistic situation, but would you like to avoid having a space between two `(%s)` when both the conda and virtualenv environments are enabled at the same time? If so, an alternative option is to use `${python_venv:+$python_venv }`.

Reply:
Thank you for your suggestion. I modified it and commited the changes.